### PR TITLE
LSD tech review

### PR DESCRIPTION
**Pull request by Loquacity**
Late-arriving tech review from @moio

### CODE REVIEW

File: modules/large-deployments/pages/hub-namespaces.adoc [hunk 1]
```diff
@@ -12,22 +12,22 @@ Hub supports three different namespaces:
 * The ``hub`` namespace is used to target the Hub XMLRPC API Server.
 It supports all the methods that are available on a {productname} Server XMLRPC API as well as Hub-specific XMLRPC endpoints.
 Hub-specific methods are primarily related to authentication.
-* The ``unicast`` namespace is used to target a single server registered in the hub.
+* The ``unicast`` namespace is used to target a single peropheral server registered in the hub.
```
Comment (keichwa) [inline modules/large-deployments/pages/hub-namespaces.adoc:15]:
```suggestion
* The ``unicast`` namespace is used to target a single peripheral server registered in the hub.
```

File: modules/large-deployments/pages/hub-namespaces.adoc [hunk 2]
```diff
@@ -12,22 +12,22 @@ Hub supports three different namespaces:
...
+* The ``multicast`` namespace is used to target multiple peripheral servers registered in the hub.
 It redirects any call transparently to all the specified servers and returns the results in the form of a ``map``.
 
 
 Some important considerations for hub namespaces:
 
-* You can obtain ``hubKey`` using different authentication methods.
-For more information, see xref:large-deployments:hub-auth.adoc[].
 * Individual server IDs can be obtained using ``client.hub.listServerIds(hubKey)``.
 * The ``unicast`` namespace assumes all methods receive ``hubKey`` and serverID as their first two parameters, then any other parameter as specified by the regular Server API
```
Comment (keichwa) [inline modules/large-deployments/pages/hub-namespaces.adoc:24]:
```suggestion
* The ``unicast`` namespace assumes all methods receive ``hubKey`` and serverID as their first two parameters, then any other parameter as specified by the regular Server API.
```

File: modules/large-deployments/pages/hub-namespaces.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/large-deployments/pages/hub-namespaces.adoc:15]: :woman_facepalming:

**Comment by moio** [comment]
> Late-arriving tech review from @moio

Duh, I'm a Java programmer, that was not late, it was [Just-In-Time](https://en.wikipedia.org/wiki/Just-in-time_compilation)! :laughing:

File: modules/large-deployments/pages/hub-auth.adoc [hunk 3]
```diff
@@ -1,12 +1,12 @@
 [[lsd-hub-auth]]
-= Hub Authentication Modes
+= Hub API Authentication Modes
```
Comment (moio) [inline modules/large-deployments/pages/hub-auth.adoc:2]:
```suggestion
= Hub XMLRPC API Authentication Modes
```

File: modules/large-deployments/pages/hub-namespaces.adoc [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [[lsd-hub-namespaces]]
-= Hub Namespaces
+= Hub API Namespaces
```
Comment (moio) [inline modules/large-deployments/pages/hub-namespaces.adoc:2]:
```suggestion
= Hub XMLRPC API Namespaces
```

File: modules/large-deployments/pages/multi-server.adoc [hunk 5]
```diff
@@ -5,6 +5,6 @@ If you need to manage a large number of clients, in most cases you can do so wit
 However, if you need to manage tens of thousands of clients, you might find it easier to use multiple {productname} Servers, in a hub, to manage them.
 
 {productname} Hub is a collection of components to help you manage very large deployments.
```
Comment (moio) [inline modules/large-deployments/pages/multi-server.adoc:7]:
```suggestion
{productname} Hub is a collection of components to help you manage very large deployments.
```

Hub is (currently) just one component: the XMLRPC API. Going forward more will be added, but we are not there yet. For now I think we should just call it "a topology".

**Comment by Loquacity** [comment]
> > Late-arriving tech review from @moio
> 
> Duh, I'm a Java programmer, that was not late, it was [Just-In-Time](https://en.wikipedia.org/wiki/Just-in-time_compilation)!

:woman_facepalming:

File: modules/large-deployments/pages/multi-server.adoc (hunk 5 above)
Comment (moio) [reply modules/large-deployments/pages/multi-server.adoc:7]: I think it's OK at least for the time being. Thanks.

**Pull request merged**